webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
+
+module.exports = {
+  entry: __dirname + '/src/main.jsx',
+  output: {
+    path: __dirname + '/build/js',
+    filename: '[name].bundle.js'
+  },
+  externals: {
+    'react': 'React',
```

Comment:
I noticed a pretty huge difference (about 60% improvement) when using externals. Also makes it simple to switch over to a CDN in the future as per @toolness 's note.

Without externals:
```bash
main.bundle.js – 929k
```

With externals:
```bash
react.min.js – 132k
main.bundle.js – 240k
```